packages/vite/src/node/plugins/resolve.ts

```diff
@@ -42,7 +43,14 @@ import {
 import { optimizedDepInfoFromFile, optimizedDepInfoFromId } from '../optimizer'
 import type { DepsOptimizer } from '../optimizer'
 import type { SSROptions } from '..'
-import type { PackageCache, PackageData } from '../packages'
+import {
+  findPackageJson,
+  isNamedPackage,
+  isWorkspaceRoot,
+  loadNearestPackageData,
+  PackageCache,
+  PackageData
+} from '../packages'
 import { loadPackageData, resolvePackageData } from '../packages'
```

Comment:
will need to update this to fix the lint failure:
```suggestion
import type {   PackageCache, PackageData } from '../packages'
import {
  findPackageJson,
  isNamedPackage,
  isWorkspaceRoot,
  loadNearestPackageData,
  loadPackageData,
  resolvePackageData,
} from '../packages'
```

Reply:
Yeah thanks, it's fixed in #9170 already :)